lib/Common/CommonDefines.h

```diff
@@ -98,11 +98,20 @@
 #if defined(_WIN32) || defined(HAS_REAL_ICU)
 #define ENABLE_UNICODE_API 1                        // Enable use of Unicode-related APIs
 #endif
+
 // Language features
-// xplat-todo: revisit these features
-#ifdef _WIN32
+// #ifdef _WIN32 // FIXME (doilij) remove or uncomment
 #define ENABLE_INTL_OBJECT                          // Intl support
+//#define INTL_ICU 1 // FIXME (doilij) remove this (need to #define this in the source code for the IDE to know about it, but the actual def will come from the project file)
+#ifdef INTL_ICU
+//#define INTL_ICU_ALLOW_HYBRID 1 // FIXME (doilij) this macro is only needed as long as INTL_ICU is in-development.
 #endif
+#if !defined(INTL_ICU) || (defined(INTL_ICU) && defined(INTL_ICU_ALLOW_HYBRID))
+// FIXME (doilij) INTL_WINGLOB only defined this way (for convenience) as long as INTL_ICU_ALLOW_HYBRID is defined, otherwise it should just be !defined(INTL_ICU)
+#define INTL_WINGLOB 1
+#endif
+// #endif // FIXME (doilij) remove or uncomment
```

Comment:
Updated.

---
In reply to: [124411993](https://github.com/Microsoft/ChakraCore/pull/3214#discussion_r124411993) [](ancestors = 124411993)